lib/mix/tasks/gen.badges.ex

```diff
@@ -24,7 +25,7 @@ defmodule Mix.Tasks.Gen.Badges do
     |> Enum.map(
       fn x ->
         case Safira.Repo.transaction(x) do
-          {:ok, result} -> result
+          {:ok, result} -> IO.inspect(result)
```

Comment:
Should this be here?

Reply:
Its usefull if we want  to inspect  if the creation is up to standards